mpf/config_spec.yaml

```diff
@@ -299,6 +299,8 @@ coils:
     default_pulse_ms: single|template_ms|None
     default_pulse_power: single|float(0,1)|None
     default_hold_power: single|float(0,1)|None
+    pulse_hold_power: single|float|None
```

Comment:
Not sure if we need those at all anymore. I propose to add `timed_enable` as a new command to coils. We should add that to `Driver` and `coil_player`. Guess we could call it a in few places such as drop targets or ball devices.

Reply:
Can you clarify how the `timed_enable` would work? Would it use the `default_hold_power`, assuming that a given coil is not going to want both enable functionality _and_ pulse-with-hold functionality (or that it would use the same power in both situations)?

Reply:
My idea would be:

A new method `timed_enable` similar to `enable` in driver (https://github.com/missionpinball/mpf/blob/dev/mpf/devices/driver.py#L193). 
```
    def timed_enable(self, enable_duration_ms, pulse_ms: int = None, pulse_power: float = None, hold_power: float = None, max_wait_ms: int = None):
```
I would make `enable_duration_ms` a required parameter here. I guess I would also include the pulse inside here (otherwise it should be called `hold_duration_ms`). This would then be called from coil_player or other devices. Not sure if we would need a default for this.

This method would call into the platform and catch `NotImplementedError` or alternatively check in `features` if this is supported. If not we would emulate it using a delay. 

Thoughts? I'm open to other suggestions.

Reply:
Interesting approach! Definitely makes sense for coil_player scenarios. For other scenarios where this behavior would be used in place of a default pulse (like a drop target reset or diverter), would we check for `timed_enable()` before using `pulse()`, or configure the coil somehow to use the timed_enable _instead_ of pulse?

Reply:
I would make this an explicit option in drop targets. Might actually become the default though in this case.